### PR TITLE
DP-4048 pattern docs

### PR DESCRIPTION
**Pull request by gender404**
Updated the Contact List Organism and Image Atom with a sample of the final Pattern Documentation to that will be added to all Patterns.

I also add a fix to remove unused JS files and some updated styling of the Pattern Info window.

https://jira.state.ma.us/browse/DP-4048

### CODE REVIEW

File: styleguide/source/_patterns/03-organisms/by-author/contact-list-without-accordion.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Contact List without accordion
+---
```
Comment (jesconstantine) [inline styleguide/source/_patterns/03-organisms/by-author/contact-list-without-accordion.md]: I see title removed in the new format for image and contact list docs, was this left in intentionally?

Comment (gender404) [reply styleguide/source/_patterns/03-organisms/by-author/contact-list-without-accordion.md]: Nope.  I just missed that one

**Comment by legostud** [comment]
Nope. I just missed it.

**Comment by jesconstantine** [comment]
Looks great to me, and in line with requests / comments from Mass.gov devs.  Thanks @legostud !

**Pull request merged**